backend/Gemfile

```
@@ -36,13 +36,28 @@ gem "bootsnap", require: false
+  gem "dotenv-rails"
 end
 
 group :development do
   # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
   # gem "spring"
+  gem "rubocop", require: false
+  gem "rubocop-performance", require: false
+  gem "rubocop-rails", require: false
+  gem "rubocop-rspec", require: false
+  gem "rubocop-rspec_rails", require: false
+  gem "rubocop-factory_bot", require: false
```

RuboCop (1.69.1)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `rubocop-factory_bot` should appear before `rubocop-rspec_rails`.

(Bundler/OrderedGems)